packages/functional-tests/tests/oauth/loginHint.spec.ts

```diff
@@ -89,28 +77,30 @@ test.describe('severity-2 #smoke', () => {
       emails,
     }) => {
       const [email, loginHintEmail] = emails;
+
       await target.createAccount(email, PASSWORD);
       await target.createAccount(loginHintEmail, PASSWORD);
-
       // Create a cached login
       await relier.goto();
       await relier.clickEmailFirst();
       await login.login(email, PASSWORD);
+
       expect(await relier.isLoggedIn()).toBe(true);
+
       await relier.signOut();
 
       // login_hint takes precedence over the signed-in user
       await relier.goto(`login_hint=${loginHintEmail}`);
       await relier.clickEmailFirst();
 
       // Email is prefilled
-      await expect(await login.getPrefilledEmail()).toEqual(loginHintEmail);
+      expect(await login.getPrefilledEmail()).toEqual(loginHintEmail);
       expect(await login.enterPasswordHeader()).toEqual(true);
 
       await login.useDifferentAccountLink();
 
       // Email first page has email input prefilled
-      await expect(await login.getEmailInput()).toEqual(loginHintEmail);
+      expect(await login.getEmailInput()).toEqual(loginHintEmail);
```

Comment:
Unrelated, but looks like the email and loginHintEmail don't get cleaned up after.

Reply:
😮  In the emails fixture I can see that the `target.authClient.accountDestroy` gets called for both emails. Is there something else required for cleanup?